packages/api-client/src/client/ClientAPI.ts

```diff
@@ -157,6 +160,28 @@ export class ClientAPI {
     await this.client.sendJSON<PreKeyBundle>(config, true);
   }
 
+  /**
+   * Will delete keypackages for an MLS capable client
+   * @see https://nginz-https.elna.wire.link/v4/api/swagger-ui/#/default/delete_mls_key_packages_self__client_
+   * @param {string} clientId The client to delete the key packages for
+   * @param {string[]} keyPackageRefs The key package references to delete
+   */
+  public async deleteMLSKeyPackages(clientId: string, keyPackageRefs: string[]) {
+    const maxKeyPackages = 1000;
```

Comment:
**question**: Is that a restriction defined by BE?

Reply:
Yes it is